src/main/java/net/earthcomputer/clientcommands/Configs.java

```diff
@@ -176,4 +176,10 @@ public enum PacketDumpMethod {
 
     @Config
     public static int maximumPacketFieldDepth = 10;
+
+    @Config(temporary = true, setter = @Config.Setter("setMinimumReplyDelaySeconds"))
+    public static float minimumReplyDelaySeconds = 0.5f;
+    public static void setMinimumReplyDelaySeconds(float minimumReplyDelaySeconds) {
+        Configs.minimumReplyDelaySeconds = Math.clamp(minimumReplyDelaySeconds, 0.0f, 10.0f);
```

Comment:
You should use the constant you made in `ReplyCommand` instead of hardcoding 10

Reply:
I agree, thought I replaced it